src/main/java/edu/harvard/iq/dataverse/export/dublincore/DublinCoreExportUtil.java

```diff
@@ -172,7 +172,11 @@ private static void createOAIDC(XMLStreamWriter xmlw, DatasetDTO datasetDto, Str
         
         writeFullElementList(xmlw, dcFlavor+":"+"language", dto2PrimitiveList(version, DatasetFieldConstant.language));        
         
-        writeFullElement(xmlw, dcFlavor+":"+"date", dto2Primitive(version, DatasetFieldConstant.productionDate));  
+        String date = dto2Primitive(version, DatasetFieldConstant.productionDate);
+        if (date == null) {
+            date = datasetDto.getPublicationDate();
+        }
+        writeFullElement(xmlw, dcFlavor+":"+"date", date);  
```

Comment:
Do we want to do something similar around line 125 above for dcTerms?

Reply:
@tcoupin, just wanted to make sure that you saw this question/comment. thanks!

Reply:
Sorry, I come back from vacation, I didn't see this notification!

I thinks it's a good idea since thedefinition of this field is:

> A point or period of time associated with an event in the lifecycle of the resource.

https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/date